x/programs/rust/wasmlanche-sdk/src/types.rs

```diff
@@ -9,8 +10,8 @@ pub type Gas = u64;
 
 /// A struct that enforces a fixed length of 32 bytes which represents an address.
 #[cfg_attr(feature = "debug", derive(Debug))]
-#[derive(Clone, Copy, PartialEq, Eq, BorshSerialize, BorshDeserialize, Hash)]
-pub struct Address([u8; Self::LEN]);
+#[derive(Clone, Copy, PartialEq, Eq, borsh::BorshSerialize, BorshDeserialize, Serialize, Hash)]
+pub struct Address(#[serde(serialize_with = "<[_]>::serialize")] [u8; Self::LEN]);
```

Comment:
Should this be limited to `cfg(test)`?

Reply:
Sure I can do that. Not for `#[cfg(test)]` though because it doesn't support inter-crate compilation sadly, that could be the `test-utils` feature though